src/Licenses.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lendable\ComposerLicenseChecker;
+
+/**
+ * @implements \IteratorAggregate<int, string>
+ */
+final class Licenses implements \IteratorAggregate
+{
+    /**
+     * @param list<non-empty-string> $licenses
+     */
+    public function __construct(
+        private readonly array $licenses,
+    ) {
+    }
+
+    public function toString(): string
+    {
+        return \implode(', ', $this->licenses);
+    }
+
+    public function isEmpty(): bool
```

Comment:
Sorry I didn't mention before, but perhaps this is `isUnlicensed`?

Reply:
For me it sounds a bit counter-intuative, because package can be unlicensed, that's right. But `Licenses` is a collection of licenses. 

Does it make sense to call it a License "licensed" or "unlicensed" ?

If you still see it fits I will rename, let me know.

Reply:
you know what, I can introduce `isUnlicensed()` method into `Package` class 🙂

Reply:
done in c3ab5a63fa8bf311d07560735c80379107091d2e

Reply:
I see your point, your suggestion makes sense.